src/views.rs

```diff
@@ -82,13 +83,16 @@ pub struct EncodableCrateOwnerInvitation {
     pub crate_id: i32,
     #[serde(with = "rfc3339")]
     pub created_at: NaiveDateTime,
+    #[serde(with = "rfc3339")]
+    pub expires_at: NaiveDateTime,
 }
 
 impl EncodableCrateOwnerInvitation {
     pub fn from(
         invitation: CrateOwnerInvitation,
         inviter_name: String,
         crate_name: String,
+        config: &Config,
```

Comment:
do we need anything else from the `Config` in here? I'm wondering if just passing `expires_at: NaiveDateTime` would be sufficient.

Reply:
Addressed!